Makefile

```diff
@@ -16,6 +16,24 @@ install_macos:
 	@conda install --file requirements-thirdparty-macos.txt --yes
 	@pip install -U -e .
 
+# Set up bioconda
+setup_bioconda:
+	@echo "Setting up bioconda"
+	@conda config --add channels bioconda
+	@conda config --add channels conda-forge
+	@conda config --add channels defaults
```

Comment:
Cant't we avoid the non-free defaults channel?

Reply:
Hmm… possibly

Reply:
https://github.com/pyani-plus/pyani-plus/blob/main/CONTRIBUTING.md does - that's what I've been running.